package.json

```diff
@@ -21,14 +21,14 @@
     "license-checker": "^25.0.1",
     "lodash": "^4.17.21",
     "semver": "^7.3.5",
-    "spdx-license-ids": "^3.0.9",
+    "spdx-license-ids": "^3.0.10",
```

Comment:
This looks sus. The Git repository has it at version `3.0.9`: https://github.com/jslicense/spdx-license-ids/blob/9d262560a6bffdf1154c858e90faad2a09d1f78b/package.json#L3

Reply:
Ah it's in a tag though: https://github.com/jslicense/spdx-license-ids/blob/v3.0.10/package.json

Reply:
https://github.com/jslicense/spdx-license-ids/issues/28